pkg/interface/src/views/apps/chat/components/chat-editor.js

```diff
@@ -172,6 +172,7 @@ export default class ChatEditor extends Component {
             fontSize="14px"
             style={{ width: '100%', background: 'transparent', color: 'currentColor' }}
             placeholder={inCodeMode ? "Code..." : "Message..."}
+            autoComplete={new RegExp(/Chrome/).test(navigator.userAgent) ? 'disabled' : 'off'}
```

Comment:
I think this doesn't work because Chrome [uses nearly the same user agent string as Safari](https://chromium.googlesource.com/chromium/src/+/master/docs/ios/user_agent.md). Try catching `CriOS` instead.

Reply:
CriOS, desTROyer of wORldS